src/routers/handlers/check_details/index.ts

```diff
@@ -23,7 +23,7 @@ export class CheckDetailsHandler extends GenericHandler<CheckDetailsViewData> {
         const baseViewData = super.getViewData(req);
 
         const details = getPresenterAccountDetails(req);
-        if (details === undefined) {
+        if (details.userId === undefined) {
             throw new Error("Presenter account details not found in session.");
         }
```

Comment:
This breaks the code. If the presenter account details are not in the session the details object will be undefined and this call will lead to an exception. being thrown. 
It was fine how it was and there is no need to change it.